Sniffs/Commenting/PropertyCommentSniff.php

```diff
@@ -94,7 +94,9 @@ protected function processTokenWithinScope(
             array(T_DOC_COMMENT_OPEN_TAG),
             $stackPtr
         );
-        if ($postComment !== false && $tokens[$postComment]['line'] === $tokens[$stackPtr]['line']) {
+        if ($postComment !== false
+            && $tokens[$postComment]['line'] === $tokens[$stackPtr]['line']
```

Comment:
I would like to see the ```&&``` in the line above, so I know, the statement will continue in the next line.